README.md

```diff
@@ -362,10 +362,13 @@ listen on 127.0.0.3:443) allows /etc/hosts entries to point to services.
 
 The top level `haproxy` section of the config file has the following options:
 
+* `do_checks`: whether or not Synapse will validate HAProxy config prior to writing it (default to `false`)
```

Comment:
This option seems redundant. Essentially, user should set both `candidate_config_file_path` and `check_command` in order to run per-validation, right?